spring/rest-authz-resource-server/README.md

```diff
@@ -53,7 +53,15 @@ For more details, see the Keycloak documentation about how to [create a new real
 Build and Run the Quickstart
 -------------------------------
 
-If your server is up and running, perform the following steps to start the application:
+If this is the first time you're running the quickstart application, you will need to install the parent POM and test helper in your local Maven repository. From the root of the repository, do the following:
+
+```
+mvn install
```

Comment:
```suggestion
mvn clean install
```
Is it possible to use `mvn clean install` here? Always slightly better to make sure that latest stuff is "enforced" to be build